articles/Frontend/Architecture/coding-guide.md

```diff
@@ -102,130 +102,125 @@ return {
+- [shorthand object notation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#New_notations_in_ECMAScript_2015)
+- [for ... of](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of)
+- [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)
+- [destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment)
+- [arrow function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions)
+- [import](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import)/[export](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export) only when the bundler supports it (library, side projects)
 
 The following coding rules apply on ES2015 code :
 
- - no `use strict`  anymore in modules, but it needs to be kept outside ES2015 modules.
- - Named function expressions can be replaced by the shorthand object notation `{ method(){ } }`
- - Do not use `class`, always prefer composition over inheritance. However there are a few use cases `class` would be allowed, for example to extend DOM prototypes, like `Error` to create new error types.
- - Try to use `const` by default (immutable) and `let` only for mutable variables (counters).
- - Be careful with destructuring, this can  create code difficult to read. A reviewer can ask to rewrite a destructred assignment if the produced code is cryptic.
- - Use arrow functions for lambda to avoid unnecessary lexical scopes, but avoid them for top-level factories or pure function definitions.
- - Use template literals instead of string concatenation.
- - Use destructuring and default parameters for method's `options` parameter : `function({label = "", num =0} = {}) { }`
+- no `use strict` anymore in modules, but it needs to be kept outside ES2015 modules.
+- Named function expressions can be replaced by the shorthand object notation `{ method(){ } }`
+- Do not use `class`, always prefer composition over inheritance. However there are a few use cases `class` would be allowed, for example to extend DOM prototypes, like `Error` to create new error types.
+- Try to use `const` by default (immutable) and `let` only for mutable variables (counters).
+- Be careful with destructuring, this can create code difficult to read. A reviewer can ask to rewrite a destructred assignment if the produced code is cryptic.
+- Use arrow functions for [lambda](https://en.wikipedia.org/wiki/Anonymous_function) to avoid unnecessary lexical scopes, but avoid them for top-level factories or [pure function](https://en.wikipedia.org/wiki/Pure_function) definitions.
+- Use template literals instead of string concatenation.
+- Use destructuring and default parameters for method's `options` parameter : `function({label = "", num = 0} = {}) { }`
```

Comment:
**chore:** Could you also add the rules enforced by ESLint, like `prefer rest over arguments`? It will help developers and it will avoid surprises.

Reply:
Nice idea, done.